scripts/fix-package-json.js

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs");
+const path = require("path");
+const {EOL} = require("os");
+
+const pkgPath = path.join(__dirname, "../pkg/package.json");
+const pkg = JSON.parse(fs.readFileSync(pkgPath, "utf8"));
+
+pkg.files = pkg.files.map((file) => {
+  if (file.endsWith("/")) {
+    return file + "**";
+  }
+  return file;
+});
+
+fs.writeFileSync(pkgPath, JSON.stringify(pkg, null, 2) + EOL, "utf8");
```

Comment:
Missing the new line here

Reply:
> Missing the new line here

Can you elaborate?

I thought I covered that with `EOL`.

Reply:
At the end of the code file itself, not I'm the call to `fs.write()`